src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.csproj

```diff
@@ -12,6 +12,9 @@
     <PackageTargetFramework>dotnet5.1</PackageTargetFramework>
     <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win</PackageTargetRuntime>
     <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime>
+    <DefineConstants Condition="'$(TargetGroup)'=='net46'">net46</DefineConstants>
```

Comment:
On second thought in your case it looks like it is currently only for the FrameworkName constant and one inline case for the version info. So I'm OK with keeping these as defines as I don't think it is worth splitting these into their own files.